tauri-app/package.json

```diff
@@ -50,8 +50,10 @@
     "@observablehq/plot": "^0.6.13",
     "@square/svelte-store": "^1.0.18",
     "@tauri-apps/api": "^1.5.3",
+    "@web3modal/ethers5": "^4.1.1",
     "codemirror-rainlang": "^3.0.3",
     "dayjs": "^1.11.10",
+    "ethers": "^5.7.2",
```

Comment:
Its too bad we couldn't get wagmi / viem working -- viem is sooo nice. But I'm happy to have something working for now.

Reply:
i dont think that really matters, the main issue was wagmi part of it, and i dont think just because viem is so nice we should automatically go for wagmi because it is based on viem?
i worked on wagmi for a few days, before switching to ethers, and personally ethers implementation is much simpler/cleaner compared to wagmi.
but after all they both do the same job and offer pretty much same funtionalities